activiti-api-process-model/src/main/java/org/activiti/api/process/model/payloads/TimerPayload.java

```diff
@@ -0,0 +1,154 @@
+package org.activiti.api.process.model.payloads;
+
+import java.util.Date;
+import java.util.UUID;
+
+import org.activiti.api.model.shared.Payload;
+
+public class TimerPayload implements Payload {
+
+    private String id;    
+    private Date dueDate;
+    private Date endDate;
+    private int retries;       
+    private int maxIterations;
+    private String repeat;
+    protected String activityId;    
+    protected String exceptionMessage;
```

Comment:
Please use `private` scope.

Reply:
Private scope is set for all:
https://github.com/Activiti/activiti-api/pull/128/commits/424814493381e36587d96212a6da7ba6ce5e5b16